Components/Queues/Drivers/AsyncQueueDriverInterface.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Smartbox\Integration\FrameworkBundle\Components\Queues\Drivers;
+
+/**
+ * Interface AsyncQueueDriverInterface
+ * @package Smartbox\Integration\FrameworkBundle\Components\Queues\Drivers
```

Comment:
We don't have tradition to setup the @package doc when creating new classes

Reply:
Ok,  removed the line.